docs/contributing/Tree-hygiene.md

```diff
@@ -257,6 +257,9 @@ When commenting on a PR, keep in mind the following mantra:
 
 It's better to close a PR than to leave it in limbo.
 
+In the case of brigading or other [code of conduct](../../CODE_OF_CONDUCT.md) violations,
+a regular contributor may intervene; this often involves removal of offending comments.
+
```

Comment:
More context not shown in GitHub diffs:

> When commenting on a PR, keep in mind the following mantra:
> 
> * Be polite and grateful. Graceful professionalism.
> * Explain what is happening. Explain why it is happening.
> * Provide next steps. Set expectations.
> 
> It's better to close a PR than to leave it in limbo.
> 
> In the case of brigading or other [code of conduct](../../CODE_OF_CONDUCT.md) violations,
> a regular contributor may intervene; this often involves removal of offending comments.

Reply:
This is in the section about how to *review* a PR. I don't think people brigading are going to be reviewers.